source/localizable/getting-started/quick-start.md

```diff
@@ -62,19 +62,25 @@ Serving on http://localhost:4200/
 
 (To stop the server at any time, type Ctrl-C in your terminal.)
 
-Open [`http://localhost:4200`](http://localhost:4200) in your browser of
-choice. You should see a page that says "Welcome to Ember" and not much
-else. Congratulations! You just created and booted your first Ember app.
-
-Switch to your editor and open `app/templates/application.hbs`. This is
-called the `application` template and it is always on screen while the
-user has your application loaded.
-
-In your editor, change the text inside the `<h2>` from `Welcome to
-Ember` to `PeopleTracker` and save the file. Notice that Ember detects
-the change you just made and automatically reloads the page for you in
-the background. You should see that "Welcome to Ember" has been
-replaced by "PeopleTracker".
+Open [`http://localhost:4200`](http://localhost:4200) in your browser
+of choice. You should see an Ember welcome page and not much
+else. Congratulations! You just created and booted your first Ember
+app.
+
+In your editor, create a new file at `app/templates/application.hbs`
```

Comment:
I used the generator mentioned [here](https://github.com/emberjs/guides/issues/1491#issuecomment-225395231) and it created a blank file in `app/templates/application.hbs`.

```
$ ember g template application
installing template
  create app/templates/application.hbs
```

I opted to have the reader create the file directly since it seemed simpler, but if generators are the preferred method I'll update the instructions.